docs/src/pages/components/passwordfield/examples/ValidationErrorExample.tsx

```diff
@@ -0,0 +1,14 @@
+import { Flex, PasswordField } from '@aws-amplify/ui-react';
+
+export const ValidationErrorExample = () => {
+  return (
+    <Flex gap="1rem" direction="column">
+      <PasswordField
+        label="Password"
+        defaultValue="1234"
+        hasError={true}
+        errorMessage="Password should not be 1234! 😱"
```

Comment:
Perhaps we can use a more realistic example such as "Password must contain at least one special character" or "Password length must be between 8-16 at least 8 characters". 

It would also be cool to conditionally display the error message depending on whether the inputted password meets the criteria, which would also reflect a more real world use case.